pkg/broker/ingress/handler.go

```diff
@@ -73,7 +73,7 @@ func (h *handler) receive(ctx context.Context, event cev2.Event) protocol.Result
 	// TODO(liu-cong) route events based on broker.
 
 	if err := h.decouple.Send(ctx, event); err != nil {
-		logging.FromContext(ctx).Desugar().Error("Error publishing to PubSub", zap.String("event", event.String()), zap.Error(err))
+		logging.FromContext(ctx).Error("Error publishing to PubSub", zap.String("event", event.String()), zap.Error(err))
```

Comment:
Shouldn't this also have Desugar()? Or is this logger already desugared coming from context?

Reply:
I switched to `knative.dev/eventing/pkg/logging` which provides a desugared logger.